src/app/pages/category/category-image/category-image.component.ts

```diff
@@ -31,4 +33,8 @@ export class CategoryImageComponent implements OnChanges {
       this.categoryImageUrl = `${this.category.images[0].effectiveUrl}`;
     }
   }
+
+  setCategoryImageName() {
+    this.categoryImageName = `${this.category.images[0]?.name}`;
```

Comment:
This will render the following image tag
```
<img loading="lazy" itemprop="image" src="https://jmetznernb.localdev.intershop.de:9443/INTERSHOP/static/WFS/inSPIRED-inTRONICS-Site/rest/inSPIRED/en_US/M/M_201807195_front.jpg" alt="front M">
```

Maybe the category name is more descriptive?